packages/lib-utils/src/components/table/VirtualizedTable.tsx

```diff
@@ -244,7 +244,8 @@ const VirtualizedTable: React.FC<VirtualizedTableProps<AnyObject>> = ({
             ))) || (
             <Tbody>
               {data.map((item, index) => (
-                <Tr>
+                // eslint-disable-next-line react/no-array-index-key
+                <Tr key={`row-${index}`}>
```

Comment:
@fhlavac we really shouldn't be using indexes for `key`. See https://reactjs.org/docs/lists-and-keys.html#keys.

It would be best to use ID's from the data if possible.